### DISCUSSION TITLE
Error creating the new SMP ServiceGroup for participant

### DISCUSSION THREAD

**Discussion by alexandrzamboch** [phax/phoss-smp#235] [General]
Hi Philip,

when I am trying to create Service group I am getting this error:

Error creating the new SMP ServiceGroup for participant 'iso6523-actorid-upis::XXXX:YYYYYYYYYYY'.
Technical details: SMPSMLException - Failed to create 'iso6523-actorid-upis::XXXX:YYYYYYYYYYY' in SML
Caused by: RegistrationHookException - Could not create business iso6523-actorid-upis::XXXX:YYYYYYYYYYY in SML
Caused by: ClientTransportException - The server sent HTTP status code 401: Unauthorized

I have changed Participant ID to XXXX:YYYYYYYYYYY

Can you please let me know what is wrong and how can I correct it?

Thank you,
Regards,
Alex

**Comment by phax** [comment]
@alexandrzamboch 
Most likely you recently acquired a new Peppol SMP certificate and forgot to update it in the SMK/SML as described on https://peppol.helger.com/public/locale-en_US/menuitem-docs-peppol-cert-update ???

The second most likely case is, that this participant is already registered in the Peppol network. Check with https://peppol.helger.com/public/locale-en_US/menuitem-tools-participant

Otherwise, the following unlikely things might also be the case
* You try to access the production SML with a Peppol-Test-SMP certificate
* You try to access the test SML with a Peppol-Production-SMP certificate
* You try to access the SML with a non-Peppol-SMP-certificate
* You changed your SMP-ID

hth, Philip

**Reply by alexandrzamboch** [reply]
Thank you Philip.

**Reply by alexandrzamboch** [reply]
Hi Philip,

I have followed all steps. I have also updated Phoss SMP to the latest version. Unfortunately I still have the same problem.

I think you are right  that it is certificate related as I have changed the certificate  recently.

I can see one error in the system: The private key should be a chain of 3 certificates but it has 2 certificates. Please ensure that the respective root certificates are contained correctly!

I don't see 3 certificates in the the certificate I got from the certification authority, there is only one.

Can you please advice what to do next?

Thank you,
Regards,
Alex

**Reply by phax** [reply]
Hi Alex,
I hope this issue was resolved in the meantime.
If not, please my SMP guide on certificate setup: https://github.com/phax/phoss-smp/wiki/Certificate-setup
hth, Philip
